src/components/Picker/Card.tsx

```diff
@@ -19,13 +19,13 @@ class Card extends React.Component<Props, {}> {
     return (
       <div>
         <div className={ cardBackground }>
-          <span><img className={ style.avatarImage + ' ' + style.cardElem } src={ this.props.image } alt="not found" /></span>
```

Comment:
@weegrom Hardcoded text like "not found" here and the one I mentioned above is not what we want. I think we should make this props so that we can do localization in the future, that's what we had discussed earlier. What do you think?

Reply:
@harsimranb  I 100% agree